vendors/laiier/models/severn_wld/model.yaml

```
@@ -0,0 +1,49 @@
+# Commercial name of the model
+name: Severn WLD™
+# Functional description of the product. Maximum 500 characters.
+description: Severn WLD™ is a wireless smart LoRaWAN water leak detection system using an adhesive, printed sensor.
+# Logo of the device
+logo: Severn_wld.jpg
+# ID(s) of the profile that defines the LoRaWAN characteristics of this model.
+deviceProfileId:
+  - laiier_RFGroup1_1.0.3a_classA
+  - laiier_RFGroup2_1.0.3a_classA
+
+# You may optionally customize any of the following settings to override the generic value set in LoRaWAN device profiles associated with your model. Leave empty if you want to keep the Device Profile settings.
+# Maximum device TX Conducted output power in dBm.
+maxTxPower:
+# Minimum device TX Conducted output power in dBm.
+minTxPower:
+# Maximum device TX Radiated output power in dBm.
+maxTxEIRP:
+# Minimum device TX Radiated output power in dBm.
+minTxEIRP:
+# Typical mobility profile of the device. Possible values are 'near_static' (also valid for static devices), 'walking_speed', 'vehicular_speed'
+# or 'random' (not known, changes over time).
+motionIndicator:
+
+# Is your device certified by the LoRa Alliance? Possible values: true, false.
+LoRaWANCertified:
+# Always mandatory: `<vendorId>:<modelName>:<modelVersion>` (You should have only one per model)
+# Be careful: Once you chose a model, it cannot be changed later
+modelId: laiier:severn_wld:1
```

underscore is not accepted in modelId.
```suggestion
modelId: laiier:severn-wld:1
```

Please rename the directory under /models also. 
The underscore is kept to identify another version of the same device.